### ISSUE TITLE
Pretrained Model for Keyphrase Extraction

### ISSUE THREAD

**Issue by whoisjones** [flairNLP/flair#1647]
Since we've integrated Keyphrase Detection Datasets it might a cool to have pretrained model for this. As a foundation we can either use Datasets from #1621 or #1646.
I would train some models and post the results here to check if it makes sense to integrate it.

**Comment by whoisjones** [comment]
according to midas-research SciBert gives (almost) the best results among all three keyphrase datasets in flair (metric: f1-score):

||INSPEC|SEMEVAL2010|SEMEVAL2017|
|---|---|---|---|
|midas|0.593|0.357|0.521|
|our model||0.351|0.444|

**Comment by alanakbik** [comment]
What are the embeddings used in our model?

**Comment by whoisjones** [comment]
SciBert, here is also the data from midas:
![image](https://user-images.githubusercontent.com/38491117/83887287-87707700-a748-11ea-947f-c67e525acd0c.png)

**Comment by alanakbik** [comment]
Model added to Flair in #1689

**Comment by djstrong** [comment]
@whoisjones How the model was trained? The scibert was finetuned in the process? What head is used (linear, rnn, crf)?
